packages/cli-kit/src/monorail.ts

```diff
@@ -76,6 +76,7 @@ export interface Schemas {
       // Environment
       env_ci?: Optional<boolean>
       env_ci_platform?: Optional<string>
+      env_package_manager_workspaces?: Optional<boolean>
```

Comment:
I think we have to create a [new version of the schema](https://github.com/Shopify/monorail/blob/42884ff91db940665b4263277d30cccf04f1daa8/schemas/app_cli3_command/1.1.yml)

Reply:
Do we? I was under the impression that if we add new optional fields we don't need to bump the schema 🤔

Reply:
Wouldn't that statement hold true for other attributes like [env_ci](https://github.com/Shopify/monorail/blob/42884ff91db940665b4263277d30cccf04f1daa8/schemas/app_cli3_command/1.1.yml#L259). Do you know @gonzaloriestra ?

Reply:
We don't need to bump a major version of the schema, but I think a minor one is required: https://production-platform.docs.shopify.io/understanding/monorail/schemas/intro#versioning-rules